tests/azure/util_test.go

```diff
@@ -0,0 +1,461 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	git2go "github.com/libgit2/git2go/v31"
```

Comment:
We can’t have this in flux CLI, please create a separate package with its own go.mod inside the test dir.

Reply:
Done